docs/user/config.md

```diff
@@ -19,3 +19,5 @@ through environment variables.
 | `PROTEUS_ASYNC_TEST_BLOCKING`      | 0 or 1 (default: 0)                                                            | Make asynchronous compilation blocking for testing                                                             |
 | `PROTEUS_TRACE_OUTPUT`             | 0 or 1 (default: 0)                                                            | Print trace output in stdout (shows information on Proteus specialization)                                     |
 | `PROTEUS_OPT_PIPELINE`             | String (default: None)                                                         | String describing a middle-end 'opt' pipeline (e.g., `default<O2>`) , when empty optimization defaults to `O3` |
+| `PROTEUS_OPT_LEVEL`               | '0', '1', '2', '3' (default: '3')                                                         | Option defining a default a middle-end 'opt' pipeline (e.g. 's' will use the 'Os' pipeline) , when empty optimization defaults to `O3` |
+| `PROTEUS_CODE_GEN_OPT_LEVEL`               | '0', '1', '2', '3', 's', 'z' (default: '3')                                                         | Option defining a default a back-end 'llc' pipeline (e.g. '1' will use the 'O1' pipeline) , when empty optimization defaults to `O3` |
```

Comment:
Shouldn't this be the other way around? `PROTEUS_OPT_LEVEL` goes 0,1,2,3,s,z while `PROTEUS_CODEGEN_OPT_LEVEL` goes 0,1,2,3 only?